xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -297,19 +297,21 @@ abstract static class Builder {
       // Private, use one of the static factory methods instead.
       protected abstract Builder clusterType(ClusterType clusterType);
 
-      // Private, use roundRobinLbPolicy() or ringHashLbPolicy(long, long).
-      protected abstract Builder lbPolicy(LbPolicy lbPolicy);
+      protected abstract Builder lbPolicyConfig(ImmutableMap<String, ?> lbPolicyConfig);
 
       Builder roundRobinLbPolicy() {
```

Comment:
I'd expect these methods to go away. Were they converted because they are used in tests?